src/components/ImageUploader/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as MImageUploader } from './src/ImageUploader.vue';
+export { default as MImagePicker } from './src/ImagePicker.vue';
+export { default as MImageSelection } from './src/ImageSelection.vue';
```

Comment:
I don't think we want to export these to be available as public components. They aren't documented or recommended for usage. We'd likely break these down further in the future to be even more atomic, revealing these now isn't necessary.